package.json

```diff
@@ -44,11 +44,15 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-matrix-org": "^0.4.0",
     "jest": "^27.5.1",
+    "react": "17.0.2",
```

Comment:
```suggestion
```

Reply:
it is used by the test currently, so I think we need it.